docs/reference/content/guides/querying.md

```diff
@@ -0,0 +1,250 @@
+the query results in different ways.
+
+### Projections
+
+[Projections]({{< docsref "reference/method/db.collection.find/#projections" >}}) allow you to return only a subset of the fields in a 
+document.  This is useful when you need to only return a smaller view of a larger object.  Borrowing from the [unit tests]({{< srcref 
+"morphia/src/test/java/org/mongodb/morphia/TestQuery.java" >}}), this is an example of this feature in action:
+
+```java
+ContainsRenamedFields user = new ContainsRenamedFields("Frank", "Zappa");
+getDs().save(user);
+
+ContainsRenamedFields found = getDs().find(ContainsRenamedFields.class).retrievedFields(true, "first_name").get();
+Assert.assertNotNull(found.firstName);
+Assert.assertNull(found.lastName);
+
+found = getDs().find(ContainsRenamedFields.class).retrievedFields(true, "firstName").get();
+Assert.assertNotNull(found.firstName);
+Assert.assertNull(found.lastName);
+```
+
+As you can see here, we're saving this entity with a first and last name but our query only returns the first name (and the _id value) in
+ the returned instance of our type.  It's also worth noting that this project works with both the mapped document field name 
+ `"first_name"` and the Java field name `"firstName"`.
+ 
+{{% note class="important" %}}
+While projections can be a nice performance win in some cases, it's important to note that this object can not be safely saved back to 
+ MongoDB.  Any fields in the existing document in the database that are missing from the entity will be removed if this entity is
+  saved. For example, in the example above if `found` is saved back to MongoDB, the `last_name` field that currently exists in the database 
+  for this entity will be removed.
+{{% /note %}}
+
+### Limiting and Skipping
+
+Pagination of query results is often done as a combination of skips and limits.  Morphia offers `Query.limit(int)` and `Query.offset(int)` 
+for these cases.  An example of these methods in action would look like this:
+
+```java
+datastore.createQuery(Person.class)
+    .offset(1)
+    .limit(10)
+    .asList()
+```
+
+This query will skip the first element and take up to the next 10 items found by the query.  There's a caveat to using skip/limit for 
+pagination, however.  See the [skip]({{< docsref "reference/method/cursor.skip" >}}) documentation for more detail.
+
+### Ordering
+
+Ordering the results of a query is done via [`Query.order(String)`](/javadoc/org/mongodb/morphia/query/Query.html#order-java.lang.String-)
+.  The javadoc has complete examples but this String consists of a list of comma delimited fields to order by.  To reverse the sort order
+ for a particular field simply prefix that field with a `-`.  For example, to sort by age (youngest to oldest) and then income (highest 
+ to lowest), you would use this:
+ 
+```java
+query.order("age,-income");
+```
+
+### Tailable Cursors
```

Comment:
Sure.  I'll take out all "window dressing" around it since if readers want to see it all they can look at the test its taken from.